packages/next/src/build/index.ts

```diff
@@ -1108,7 +1108,8 @@ export default async function build(
                 config,
                 distDir,
                 pageKeys,
-                pageInfos: [],
+                // TODO: Is this wrong?
+                pageInfos: new Map(),
```

Comment:
This can't be a `Map` when using `Worker` as it's not serializable by default.

Reply:
We could omit this here and instead have a default arg value in the function since we aren't passing actual page info through here.